html/secondary-header.html

```diff
@@ -0,0 +1,21 @@
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<link href="css/styles.css" rel="stylesheet">
+
+<div id="nav-secondary">
+  <nav class="navbar nav-main navbar-default" style="background-color: #000;">
+    <ul class="nav navbar-nav navbar-custom nav-ul-second">
+      <li><a class="navlink" href="http://polar.usc.edu/Polar/banana/#/dashboard">Banana</a></li>
+      <li><a class="navlink" href="http://polar.usc.edu/Polar/facetview/">FacetView</a></li>
+    </ul>
+  </nav>
+</div>
+
+<!-- Use this script to generate secondary header -->
```

Comment:
@chrismattmann, it's a script that Janet/Titus can use to render the header wherever they need to. Thought it'd be useful so I put it in there. (not used there - just a sort of guide. Can move to anywhere if you want though)